src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/Models/ReindexJobRecord.cs

```diff
@@ -58,6 +58,9 @@ protected ReindexJobRecord()
 
         public ConcurrentDictionary<ReindexJobQueryStatus, byte> QueryList { get; private set; } = new ConcurrentDictionary<ReindexJobQueryStatus, byte>();
 
+        [JsonProperty(JobRecordProperties.ResourceCount)]
+        public ConcurrentDictionary<string, int> ResourceCount { get; private set; } = new ConcurrentDictionary<string, int>();
```

Comment:
>ResourceCount [](start = 49, length = 13)

This is collection, something in this name should be plural :) `ResourceCounts`?

Reply:
Fixed in 7555909.